tsconfig.json

```diff
@@ -39,7 +39,10 @@
     "paths": {
       // Original: docs/node_modules/@storybook/api/node_modules/@emotion/core/types/index.d.ts
       // Since `@emotion/core` does not exist anymore, let's resolve it to `@emotion/react`.
-      "@emotion/core": ["./node_modules/@emotion/react"]
+      "@emotion/core": ["./node_modules/@emotion/react"],
+      "@/storybook-helpers": ["./storybook/src/helpers/index"]
     }
-  }
+  },
+  // @todo: remove, once storybook v8 replaced the v5 version
+  "exclude": ["./storybook"]
```

Comment:
Currently, the `storybook` folder needs to be excluded from typechecking, since the typechecker [complains](https://github.com/commercetools/ui-kit/actions/runs/10076964662/job/27858442821?pr=2864) about missing dependencies. As long as the v5 storybook version is still present and in use, all the tooling relies on the v5 storybook packages being present, means the failing can not be prevented either.
Once we're confident all v8 stories (separate branch) provide feature parity to the v5 ones, the v5 storybook packages & stories can be removed (as well as this line) and storybook can become a regular workspace.